modules/op-release-notes-1-6.adoc

```diff
@@ -306,4 +306,29 @@ Error from server (InternalError): Internal error occurred: failed calling webho
 ** For implicit params:
 *** Do not apply the `PipelineSpec` parameters to the `TaskRefs` object.
 *** Disable implicit param behavior for the `Pipeline` objects.
-// https://github.com/tektoncd/pipeline/releases/tag/v0.28.3
\ No newline at end of file
+// https://github.com/tektoncd/pipeline/releases/tag/v0.28.3
+
+[id="release-notes-1-6-3_{context}"]
+== Release notes for {pipelines-title} General Availability 1.6.3
+
+[id="fixed-issues-1-6-3_{context}"]
+=== Fixed issues
+
+* Before this update, the {pipelines-title} Operator installed pod security policies from components such as Pipelines and Triggers. However, the pod security policies shipped as part of the components were deprecated in an earlier release. With this update, the Operator stops installing pod security policies from components. As a result, the following upgrade paths are affected:
+** When you upgrade from {pipelines-shortname} 1.6.1 or 1.6.2 to {pipelines-shortname} 1.6.3, the existing pod security policies from components such as Pipelines and Triggers are deleted.
+** When you upgrade from {pipelines-shortname} 1.5.x to 1.6.3, the existing pod security policies are retained. However, cluster administrators can delete them manually.
```

Comment:
```suggestion
** Upgrading from {pipelines-shortname} 1.5.x to 1.6.3 retains the pod security policies. As a cluster administrator, delete these pod security policies by using the `[TBD]` command.
```
- You can remove "However." It creates a false opposition with the first sentence.
- Do we need to clarify whether the policies are still in-effect or tell the administrator to delete them?